erc20-load-test-tool/script/nginx/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM harbor-repo.vmware.com/app_perf/nginx:latest
+COPY nginx.conf /etc/nginx/nginx.conf
```

Comment:
Discussed with @ramkri123 . We want to use load balancer, but rather than packaging with this tool, we can make it as part of k8s service mesh deployment.

Reply:
okay